tests/std/tests/P2286R8_formatting_ranges_legacy_text_encoding/env.lst

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# This is `concepts_latest_matrix.lst` with `/execution-charset:.932` added.
+# clang is excluded since it doesn't support non-UTF-8 execution charsets.
+
+RUNALL_INCLUDE ..\prefix.lst

```

Comment:
why do we need this to have it's own custom env.lst? I get we need charset specific tests but a lot of this is repeating stuff that we could do with RUNALL_CROSLIST from one of the presets

Reply:
This file is modeled after [`tests/std/tests/P0645R10_text_formatting_legacy_text_encoding/env.lst`](https://github.com/microsoft/STL/blob/main/tests/std/tests/P0645R10_text_formatting_legacy_text_encoding/env.lst). I guess it can't use `RUNALL_CROSSLIST` (like [`tests/std/tests/P0645R10_text_formatting_utf8/env.lst`](https://github.com/microsoft/STL/blob/main/tests/std/tests/P0645R10_text_formatting_utf8/env.lst)) because it needs to exclude Clang, which doesn't support `/execution-charset:.932`.

Reply:
yeah I'm OK with this. If we ever need another test like this we might want to hoist this up into a common lst.